src/clr-core/internal/utils/register.ts

```diff
@@ -26,3 +29,7 @@ export function registerElementSafely(tagName: string, elementClass: any) {
     addElementToRegistry(tagName, elementClass, window.customElements);
   }
 }
+
+function isStorybook() {
+  return window.location.href.includes('localhost:6006');
```

Comment:
is this the best way to know if we're running inside storybook? it seems a little brittle...

Reply:
Yeah I didnt like this one either. I can dig a bit more. There isnt much exposed to the runtime code since they put the stories in iframes to sandbox them.